### ISSUE TITLE
Cannot get window size while resizing

### ISSUE THREAD

**Issue by faizan170** [devbookhq/splitter#13]
I am using this library in react with canvas. I need to update canvas width when resizing windows using this package.
But it provides on **resize start** and **resize finish** methods. Is there any way to get data while resizing window?

**Comment by faizan170** [comment]
Yes, we need to method to rerender points on canvas while resizing. So for this we need to make it real time to use that method. I tried some other options like checking width of modified elments while resizing but splitter start causing issues.

**Comment by mlejva** [comment]
I see. I don't really have a working solution or a hack right now, unfortunately. I need to think about this once I have some allocated time for this project. 
I can't right away tell you whether it would be a worthy trade off given the goal of Splitter being as little JS powered as possible.
